Automattic/jetpack
# Anchor.fm: Show single episode in player when creating episode post

#### Changes proposed in this Pull Request:

- Modifies the podcast-player block with an attribute to specify a list of selected episodes to show in the player
- Updates the /podcast-player API endpoint to accept a list of guids to search for specific episodes
- Updates the Anchor.fm single podcast episode post template to show only that episode in the podcast player

This PR uses the exploratory work in https://github.com/Automattic/jetpack/pull/18201, extracting the minimum changes needed to display a single episode podcast player when creating an episode post from Anchor.fm.

#### Jetpack product discussion

pbAPfg-PF-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:






* Past a link to create a new post that includes `anchor_podcast` and `anchor_episode` query parameters
* For example: `http://localhost:8000/wp-admin/post-new.php?anchor_episode=547d02ae-abde-462d-92af-3de55ed5bcd2&anchor_podcast=22b6608`
* You should see only the episode identified by `anchor_podcast` in the player.

#### Proposed changelog entry for your changes:
* n/a, I think since it's specifically related the Anchor.fm -> WP.com flow
